packages/math-input/src/full-math-input.stories.tsx

```diff
@@ -25,11 +25,17 @@ export const Basic = () => {
     const [keypadElement, setKeypadElement] = React.useState<KeypadAPI>();
     // Whether to use Expression or Fraction keypad
     const [expression, setExpression] = React.useState<boolean>(false);
+    // Whether to use CDOT or TIMES
+    const [times, setTimes] = React.useState<boolean>(true);
     // Whether to use v1 or v2 keypad
     const [v2Keypad, setV2Keypad] = React.useState<boolean>(true);
     // Whether the keypad is open or not
     const [keypadOpen, setKeypadOpen] = React.useState<boolean>(false);
 
+    const input = React.useRef<any>(null);
```

Comment:
Would something like this work?
```suggestion
    const input = React.useRef<typeof KeypadInput>(null);
```